jdaviz/configs/default/plugins/plot_options/plot_options.py

```diff
@@ -38,8 +46,19 @@ def _on_select_viewer_message(self, msg):
         # message from elsewhere requesting to change the selected viewer
         self.selected_viewer = msg.viewer
 
+    def _on_data_changed(self):
+        self.show_uncertainty = False

```

Comment:
Is there a reason you had to default to turning off the uncertainties when new data is added, rather than showing them for the new data as well if they were toggled on?

Reply:
Good point. I made this change before I went in and refactored the uncertainties/mask code on the backend. I can switch this to work as you described.